arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-nigeria.dts

```diff
@@ -170,6 +172,66 @@
 		compatible = "microchip,24lc256","atmel,24c256";
 		reg = <0x50>;
 	};
+
+	i2cswitch@70 {
+		compatible = "nxp,pca9546";
+		reg = <0x70>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		p0_channel_0_2: i2c@2 {
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <2>;
+			pvddcrcpu0p0@60 {
+				//PVDDCR_CPU0_VDDIO_MEM_S3_P0_CONTROLLER
+				compatible = "mps,mp2857";
+				reg = <0x61>;
+			};
+			pvddcrcpu1p0@61 {
+				//PVDDCR_CPU1_PVDDCR_SOC_P0_CONTROLLER
+				compatible = "mps,mp2857";
+				reg = <0x62>;
+			};
+			pvddiop0@63 {
+				//PVDDIO_P0_CONTROLLER
+				compatible = "mps,mp2857";
+				reg = <0x63>;
+			};
+		};
+	};
+};
+
+&i2c9 {
+	// Net name SCM_I2C<1>
+	status = "okay";
+
+	i2cswitch@70 {
+		compatible = "nxp,pca9546";
+		reg = <0x70>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		p1_channel_0_2: i2c@2 {
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <2>;
+			pvddcrcpu0p1@61 {
+				//PVDDCR_CPU0_VDDIO_MEM_S3_P1_CONTROLLER
+				compatible = "mps,mp2857";
+				reg = <0x61>;
+			};
+			pvddcrcpu1p1@62 {
```

Comment:
Nigeria VR at 0x65 is MP2869
does this driver work fir it?

Reply:
We did not find a matching driver for MP2869 in the linux kernel
The driver mp2857 seems to more compatible for the MP2869 model.
This will be tested when Nigeria is available. However , if the driver is not working properly we need to request a driver for these parts from MPS.